1_fetch/src/get_nwis_data.R

```diff
@@ -1,10 +1,10 @@
 
-download_nwis_data <- function(site_nums = c("01427207", "01432160", "01435000", "01436690", "01466500")){
+download_nwis_data <- function(site_nums){
   
   # create the file names that are needed for download_nwis_site_data
   # tempdir() creates a temporary directory that is wiped out when you start a new R session; 
   # replace tempdir() with "1_fetch/out" or another desired folder if you want to retain the download
-  download_files <- file.path(tempdir(), paste0('nwis_', site_nums, '_data.csv'))
+  download_files <- file.path("1_fetch/tmp/", paste0('nwis_', site_nums, '_data.csv'))
```

Comment:
No need to change this now, but you would also have to commit an .empty file to this directory (or check if this directory exists and create it if it doesn't) in order for the pipeline to work off the shelf for someone else.

Reply:
ahhh. forgot about this folder. yes. I'll add